### DISCUSSION TITLE
请问 想要使用 taro，一定要从零用 tarojs/cli 创建项目吗？

### DISCUSSION THREAD

**Discussion by linhaihero** [NervJS/taro#16508] [Q & A] [answered]
如题，有没有办法从vue官方创建的项目中，修改调整一下就能使用这样呢？求大佬解惑

**Comment by TheKonka** [comment, marked as answer]
vue 官方的恐怕不行，小程序项目的目录结构还是有些差异
不使用 tarojs/cli 创建，也可以在别的 taro 项目基础上进行修改

**Comment by Jeff-Tian** [comment]
对于这个场景，不知道使用 kbone 会不会更简单一些。也希望有大佬可以分享一下，有没有办法将 kbone 集成在 taro 中？

**Comment by ShaoGongBra** [comment]
duxapp这个框架可以试一下，做了很多封装，http://duxapp.cn
